doc/introduction/introducing_zephyr.rst

```diff
@@ -4,11 +4,12 @@ Introducing Zephyr
 ##################
 
 The Zephyr OS is based on a small-footprint kernel designed for use on
-resource-constrained systems: from simple embedded environmental sensors and LED
-wearables to sophisticated smart watches and IoT wireless gateways.
+resource-constrained and embedded systems: from simple embedded environmental
+sensors and LED wearables to sophisticated embedded controllers, smart
+watches and IoT wireless applications.
```

Comment:
Since I had other suggestions, I'll mention a missing comma here:

    watches, and ...